docs/community/maintainers-guide/releasing.rst

```diff
@@ -0,0 +1,224 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2020 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Releasing
+=========
+
+In general, you have two types of releases:
+
+- **Major/Minor-level release** - Often these releases add significant new
+  features and needs to go through extensive testing.
+- **Patch-level release** - Often these releases fix security issues or
+  important bugs.
+
+For major/minor-level releases, you'll often be working only with the
+``master``-branch and only make a single PyPI release. For patch-level
+releases you'll often be working with multiple maintenance branches and make
+several PyPI releases for all currently maintained versions of Invenio.
+
+Depending on the package that you are releasing, you will have to follow
+a different procedure. You might have to release:
+
+- one of the Python or JavaScript/React modules (happening rather often)
+- the Invenio framework, the ``invenio`` package (happening less often)
+
+Before you start
+----------------
+Prior to preparing a new Invenio release, please ensure you have the latest
+changes in your local environment for the maintenance branches and ``master``:
+
+1. Check that all relevant PRs have been merged. For the ``invenio`` package,
```

Comment:
Styling for bullet one is a bit off:
![Screenshot 2020-08-26 at 14 30 43](https://user-images.githubusercontent.com/1698163/91303738-b9720200-e7a8-11ea-8208-b9446fee012d.png)

Reply:
The identation in the code seems correct. If put one tab less, then the enumerated list is failing (ends and restarts after...). Same if trying 2 spaces instead of a full tab. e.g

```
/Users/ppanero/Workspace/inveniosw/invenio/docs/community/contributing/maintainers-guide/releasing.rst:39:Enumerated list ends without a blank line; unexpected unindent.
```

Reply:
I have changed, see [here](https://stackoverflow.com/a/5553270/675178)... because we ❤️ Sphinx, right? :)